test/test_logging_output_timestamps.regex

```diff
@@ -0,0 +1 @@
+'[\s,-][0-9]{10}[.][0-9]{9}' '[\s,-][0-9]{19}'\n'[\s,-][0-9]{10}[.][0-9]{9}' '[\s,-][0-9]{19}'
```

Comment:
`\s` instead of ` ` will also match other whitespace characters which seems unnecessarily loose. Was there a reason for not just using a space?

Reply:
Nope, I can change that and rerun the tests.

Reply:
no need now I was mostly just curious if there was regex trickery going on